plugins/chat/app/services/chat/mark_thread_title_prompt_seen.rb

```diff
@@ -51,7 +51,11 @@ def threading_enabled_for_channel(thread:)
 
     def create_or_update_membership(thread:, guardian:, contract:)
       membership = thread.membership_for(guardian.user)
-      membership = thread.add(guardian.user) if !membership
+      membership =
+        thread.add(
+          guardian.user,
+          notification_level: Chat::NotificationLevels.all[:normal],
```

Comment:
if create the membership on behalf of the someone, we don't want to set it to tracking by default.